pages/services/beta-storage/0.4.0-beta/index.md

```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: DC/OS Storage Service 0.4.0
 title: DC/OS Storage Service 0.4.0
-menuWeight: 0
+menuWeight: 10
```

Comment:
What does menuWeight - 10 means?

Reply:
@imaxx Here, menu weights are assigned to each file/page to show the hierarchy of occurrence in the left side of the navigation panel in an incremental way. Menu weight start from Zero to 100+
**Example:**
See the left side navigation panel on https://docs.mesosphere.com/services/cassandra/2.4.0-3.0.16/

Page - assigned menu weight

- Release Notes - 10
- Getting Started - 20 
- Operations - 40

.......